cmd/gafFilter/gafFilter.go

```diff
@@ -0,0 +1,76 @@
+// Command Group:
+
+// Output a subset of a gaf file without "NOT" entries in qualifier field
+package main
+
+import (
+	"flag"
+	"fmt"
+	"github.com/vertgenlab/gonomics/exception"
+	"github.com/vertgenlab/gonomics/fileio"
+	"github.com/vertgenlab/gonomics/ontology/gaf"
+	"log"
+	"strings"
+)
+
+type Settings struct {
+	InFile    string
+	OutFile   string
+	RemoveNot bool
+}
+
+func gafFilter(s Settings) {
+	var pass bool
+
+	gafChan, header := gaf.GoReadToChan(s.InFile)
+	out := fileio.EasyCreate(s.OutFile)
+
+	gaf.WriteHeaderToFileHandle(out, header)
+
+	for curr := range gafChan {
+		pass = true
+		if s.RemoveNot {
+			if strings.Contains(curr.Qualifier, "NOT") {
```

Comment:
could we make this work only on the column where we expect NOTs to appear instead of the whole gaf line?